guides/doc-Creating_a_Standard_Operating_Environment/master.adoc

```diff
@@ -0,0 +1,29 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: soe
+:ProductName: {ProjectServer}
+
+= {SOEDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+[part]
+include::common/modules/con_setting-up-your-project-system-management-infrastructure.adoc[]
+
+include::common/assembly_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+1]
+
+include::common/assembly_enabling-and-synchronizing-the-project-client-name-repository.adoc[leveloffset=+1]
```

Comment:
If you want this guide to make sense for non-Katello, this should be hidden.
```suggestion
ifdef::katello,satellite,orcharhino[]
include::common/assembly_enabling-and-synchronizing-the-project-client-name-repository.adoc[leveloffset=+1]
endif::[]
```

Reply:
Addressed... by removing the whole assembly! It's currently part of the installation guide. For SOE, I want to try to focus on post-installation guide tasks only. At least for now, until I change my mind. Again.